packages/cozy-client/src/models/instance.js

```diff
@@ -0,0 +1,26 @@
+import get from 'lodash/get'
+const GB = 1000 * 1000 * 1000
+const PREMIUM_QUOTA = 50 * GB
+
+const instance = {
+  //if managet url is here, then it's not selfhosted
+  isSelfHosted: instanceInfo => {
+    return get(instanceInfo, 'context.data.attributes.manager_url')
+      ? false
+      : true
+  },
+  arePremiumLinksEnabled: instanceInfo => {
+    return get(instanceInfo, 'context.data.attributes.enable_premium_links')
+      ? true
+      : false
+  },
+  isFreemiumUser: diskUsage => {
+    const quota = get(diskUsage, 'data.attributes.quota', false)
+    return parseInt(quota) < PREMIUM_QUOTA
+  },
+  getUuid: instanceInfo => {
+    return get(instanceInfo, 'instance.data.attributes.uuid')
+  }
+}
+
+export { instance }
```

Comment:
An alternative to exporting an object is to export the functions.

```
export const getUuid = instanceInfo => get(instanceInfo, 'instance.data.attributes.uuid')

export * as instance from './instance'
```

This would enable tree shaking.

or to export as default 

```
export instance from './instance'
```

I personally both of these alternative to be more predictible as user that the exporting of { instance }.

Reply:
I like the simplicity of
```jsx

export const getUuid = instanceInfo => get(instanceInfo, 'instance.data.attributes.uuid')
export * as instance from './instance'
```

Reply:
The change in triggers.js would be breaking I guess, I think it is better if done in another PR.